src/fs-access/file-save.mjs

```diff
@@ -67,6 +67,17 @@ export default async (
       excludeAcceptAllOption: options[0].excludeAcceptAllOption || false,
     }));
   const writable = await handle.createWritable();
+  // use streaming on the blob if the browser supports it
+  if ('stream' in blobOrStream) {
```

Comment:
I don't get this part. There's no `stream()` method on `ReadableStream` ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream#methods)). I may be misunderstanding what you want to do here.

Reply:
Oh, sorry. This is referring to [`Blob.stream()`](https://developer.mozilla.org/en-US/docs/Web/API/Blob/stream). Ignore my comment above.